### PR TITLE
segregate js rules for aws,google,azure,atlassian

### PR DESCRIPTION
**Pull request by hiaashna**

### CODE REVIEW

File: rules/sinks/third_parties/sdk/amazon/javascript.yaml [hunk 1]
```diff
@@ -193,3 +193,859 @@ sinks:
...
     tags:
+
+  - id: ThirdParties.SDK.Amazon.Aws.S3
+    name: Amazon Aws S3
```
Comment (dbMundada) [inline rules/sinks/third_parties/sdk/amazon/javascript.yaml:198]: S3 already present. Please delete this one.

File: rules/sinks/third_parties/sdk/amazon/javascript.yaml [hunk 2]
```diff
@@ -193,3 +193,859 @@ sinks:
...
+    domains:
+      - "aws.amazon.com/s3"
+    patterns:
+      - "@aws-sdk\\/client-s3"
```
Comment (dbMundada) [inline rules/sinks/third_parties/sdk/amazon/javascript.yaml:202]: I see some rules already present can we delete duplicate rules.

File: rules/sinks/third_parties/sdk/atlassian/javascript.yaml [hunk 3]
```diff
@@ -12,6 +12,68 @@ sinks:
...
     tags:
 
+  - id: ThirdParties.SDK.Atlassian.Confluence
+    name: Atlassian Confluence
+    domains:
```
Comment (dbMundada) [inline rules/sinks/third_parties/sdk/atlassian/javascript.yaml:17]: Can you remove the packages from `ThirdParties.SDK.Atlassian` rule which we added for other rules.
Same feedback for other files.

Comment (hiaashna) [reply rules/sinks/third_parties/sdk/atlassian/javascript.yaml:17]: removed

File: rules/sinks/third_parties/sdk/amazon/javascript.yaml (hunk 2 above)
Comment (hiaashna) [reply rules/sinks/third_parties/sdk/amazon/javascript.yaml]: removed

**Review by dbMundada — APPROVED** [review]

**Review by dbMundada — APPROVED** [review]

**Pull request merged**